assets/js/app.js

```diff
@@ -0,0 +1,51 @@
+const productName = document.getElementById('productName');
+const productPrice = document.getElementById('productPrice');
+const productType = document.getElementById('productType');
+const productImage = document.getElementById('productImage');
+const productForm = document.getElementById('productForm');
+const cartTableBody = document.querySelector('#cartTable tbody');
```

Comment:
better refrain from using global variables. we'll touch this issue soon in class

Reply:
היי שחר בנוגע להערות שלך. לפניי שהגשתי את המבחן העברתי את הקוד בצאט גיפיטי וישמתי חלק מההערות שלו. בנוסף לקראת הסוף (של הסוללה בלפטופ שלי) ניסיתי להעלות את הקוד לגיט כדיי שאוכל להמשיך את המבחן מהמחשב שהיה בכיתה. כשזה לא צלח נעזרתי בכלים באנטרנט כדיי לסיים את המבחן מהר יותר.

לא הכרתי את querySelector אך הצאט אמר שזו דרך יותר יעילה לבחור אלמנט בטבלה אז החלפתי את הgetelementbyid שלי לזה. אותו הדבר גם על פונקציות {}<=() אמר שהן אותו הדבר אך יותר יעילות לפעולות שבהן הן רשומות.